worlds/sc2/item_descriptions.py

```diff
@@ -896,7 +899,7 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     # Colossus
     item_names.WRATHWALKER_AERIAL_TRACKING: "Wrathwalker War Council upgrade. Wrathwalkers can now target air units.",
     item_names.REAVER_KHALAI_REPLICATORS: "Reaver War Council upgrade. Reaver Scarabs no longer cost minerals.",
-    # Disruptor
+    item_names.DISRUPTOR_RESTRUCTURED_THRUSTERS: "Allows the Disruptor to move while casting Purification Nova. Also allows the Disruptor to Blink.",

```

Comment:
Please include a sentence at the start "Disruptor War Council upgrade." to match the others / make it clearer in the icon repository that it's part of the set.